openstf/feature_engineering/holiday_features.py

```diff
@@ -3,19 +3,24 @@
 # SPDX-License-Identifier: MPL-2.0
 
 from datetime import timedelta, datetime
+from typing import Tuple
 
 import numpy as np
 import pandas as pd
 import holidays
 
 from openstf import PROJECT_ROOT
 
-HOLIDAY_CSV_PATH = PROJECT_ROOT / "openstf" / "data" / "dutch_holidays_2020-2022.csv"
+HOLIDAY_CSV_PATH: str = (
+    PROJECT_ROOT / "openstf" / "data" / "dutch_holidays_2020-2022.csv"
+)
 
 
-def create_holiday_functions(
-    country="NL", years=None, path_to_school_holidays_csv=HOLIDAY_CSV_PATH
-):
+def create_holiday_feature_functions(
+    country: str = "NL",
```

Comment:
@JanMaartenvanDoorn Does it makes sense to use a default argument here? Why not require to be explicit?

Reply:
Yes, in case of a notebook I think we want to have minimal configuration and we just want to apply the features.

Reply:
We are not the only people using this library. And I don't think its really too much effort to write down which country you want in a notebook. In my opinion we should not use default/optional arguments in such cases.